src/components/Editor.jsx

```diff
@@ -72,6 +72,12 @@ class Editor extends React.Component {
     } = this.state
 
     const codeMirrorInst = this.codeMirror.getCodeMirror();
+
+    // If this is the first time through, value may be set without Codemirror
+    // knowing about it. Push the value to codemirror to rectify
+    if (codeMirrorInst.getValue() === '' && value !== '') {
+      codeMirrorInst.setValue(value);
+    }
```

Comment:
Conditionals turned out to still be required, otherwise it lead to a strange bug where the first character would appear after the cursor. I nested the conditionals in a onced function, so they will only be checked the first time.